test/hotspot/jtreg/serviceability/jvmti/vthread/MethodExitTest/libMethodExitTest.cpp

```diff
@@ -452,13 +452,24 @@ VirtualThreadMount(jvmtiEnv *jvmti, ...) {
   mname = get_method_name(jvmti, jni, method);
   cname = get_method_class_name(jvmti, jni, method);
 
+  print_frame_event_info(jvmti, jni, thread, method, "VirtualThreadMount", ++vthread_mounted_count);
+
   LOG("\nHit #%d: VirtualThreadMount #%d: enabling FramePop for method: %s::%s on virtual thread: %p\n",
-         brkptBreakpointHit, ++vthread_mounted_count, cname, mname, (void*)thread);
+         brkptBreakpointHit, vthread_mounted_count, cname, mname, (void*)thread);
 
   err = jvmti->NotifyFramePop(thread, 0);
   check_jvmti_status(jni, err, "VirtualThreadMount: error in JVMTI NotifyFramePop");
 
-  print_frame_event_info(jvmti, jni, thread, method, "VirtualThreadMount", vthread_mounted_count);
+  LOG("\nHit #%d: VirtualThreadMount #%d: enabling duplicated FramePop for method: %s::%s on virtual thread: %p\n",
+         brkptBreakpointHit, vthread_mounted_count, cname, mname, (void*)thread);
+
+  err = jvmti->NotifyFramePop(thread, 0);
+  if (err == JVMTI_ERROR_DUPLICATE) {
+    LOG("NotifyFramePop at VirtualThreadUnmount event returned expected JVMTI_ERROR_DUPLICATE\n");
+  } else {
+    LOG("Failed: expected JVMTI_ERROR_DUPLICATE from NotifyFramePop at VirtualThreadUnmount event\n");
```

Comment:
Would be nice to log returned error, something like

```
    LOG("Failed: NotifyFramePop at VirtualThreadUnmount returned %s (%d) instead of expected JVMTI_ERROR_DUPLICATE\n",
           TranslateError(err), err);
```

Reply:
Thanks. Updated now.